mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -28,6 +28,7 @@
 import java.util.Map;
```

Comment:
I really think the test is not "annotation is deleted", we don't care about that, it's more "there's no way to retrieve these (outdated) annotations by any API". Deletion from database is not needed.